pullrequest.rst

```diff
@@ -47,6 +47,10 @@ Here is a quick overview of how you can contribute to CPython:
 .. [*] If an issue is trivial (e.g. typo fixes), or if an issue already exists,
        you can skip this step.
 
+.. note::
+   In order to keep the commit history intact, please avoid force-pushing to
```

Comment:
```suggestion
   In order to keep the commit history intact, please avoid squashing or amending history and then force-pushing to
```

Reply:
I'm fine with as-is; I feel the additional suggested wording  to be helpful.

Reply:
If @aeros agrees on it, too, I'm happy to adjust the wording as proposed by you.

Reply:
Yep, that looks good to me.

Reply:
To elaborate, I think it helps to clarify that it's specifically modification to the commit history that can be harmful to review, and not just force-pushing itself.

Reply:
Alright, I adjusted it accordingly. Thanks for pointing out!